### DISCUSSION TITLE
Using arbitrary group seems to break when combining with `:after` and `content`

### DISCUSSION THREAD

**Discussion by marcdix** [tailwindlabs/tailwindcss#11371] [Help]
In the following example you see two elements: One that has `group/one` and one that just has `group`. You can then see that applying `bg-yellow-200` to both works. Then you see that a pipe character is applied via `:after:content`, which only works in the second case (see rendering on right side). It doesn't work for the first case (it's not rendering the utility class at all - should generate 4 classes, but only generates 3). If you remove the single quotes in the first case, making it this `group-[.foo]/one:after:content-[|]`, you can see that the class renders (but ofc now the quotes are missing and it doesn't work either). Any ideas?
 
https://play.tailwindcss.com/rqohkGKx9p

**Comment by wongjn** [comment]
Seems like the Tailwind parser does not like the arbitrary value with quotes with an arbitrary group variant that has a modifier: https://play.tailwindcss.com/omy9velOkI
